assets/in

```diff
@@ -32,6 +32,8 @@ repository="$(jq -r '.source.repository // ""' < $payload)"
 tag="$(jq -r '.source.tag // "latest"' < $payload)"
 ca_certs=$(jq -r '.source.ca_certs // []' < $payload)
 client_certs=$(jq -r '.source.client_certs // []' < $payload)
+max_concurrent_downloads=$(jq -r '.source.max_concurrent_downloads // 3' < $payload || echo "3")
+max_concurrent_uploads=$(jq -r '.source.max_concurrent_uploads // 3' < $payload || echo "3")
```

Comment:
Why the `echo 3`?

Reply:
Default value 3

Reply:
Indeed it [does](https://stedolan.github.io/jq/manual/#Alternativeoperator://).